extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/handlers/DevClasspathStaticHandler.java

```diff
@@ -0,0 +1,178 @@
+package io.quarkus.vertx.http.runtime.handlers;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.util.Set;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.vertx.http.runtime.GeneratedStaticResourcesRecorder;
+import io.vertx.core.Handler;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.http.impl.MimeMapping;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.StaticHandler;
+
+/**
+ * {@link StaticHandler} implementation to handle static resources using the Classpath.
+ * This is meant to be used on {@link io.quarkus.runtime.LaunchMode#DEVELOPMENT} mode.
+ */
+public class DevClasspathStaticHandler implements Handler<RoutingContext> {
+
+    private static final Logger LOG = Logger.getLogger(DevClasspathStaticHandler.class);
+    private static final Set<HttpMethod> ALLOWED_HTTP_METHODS = Set.of(HttpMethod.GET, HttpMethod.HEAD, HttpMethod.OPTIONS);
+    private static final int HTTP_STATUS_OK = 200;
```

Comment:
Don't we have these constant somewhere else?

Reply:
I made a search for  `= 200` in the project but there is no option. I have access to `package io.netty.handler.codec.http.HttpResponseStatus`, makes sense to use it?